tools/Az.Tools.Predictor/Az.Tools.Predictor/Az.Tools.Predictor.psd1

```diff
@@ -29,15 +29,15 @@ CompanyName = 'Microsoft Corporation'
 Copyright = 'Microsoft Corporation. All rights reserved.'
 
 # Description of the functionality provided by this module
-Description = 'Microsoft Azure PowerShell: Provide prediction while user typing Azure PowerShell commands'
+Description = 'Microsoft Azure PowerShell Predictor: Provide prediction while user types Azure PowerShell commands'
 
 # Minimum version of the PowerShell engine required by this module
 PowerShellVersion = '7.1'
 
 # Modules that must be imported into the global environment prior to importing this module
-#RequiredModules = @(@{ModuleName = 'PSReadLine'; ModuleVersion = '2.0'; })
+RequiredModules = @(@{ModuleName = 'PSReadLine'; ModuleVersion = '2.1.0'; }, @{ModuleName = 'Az'; ModuleVersion = '3.0.0'; })
```

Comment:
The PSReadLine version this module depends on should be `2.2.0-beta1` eventually, because our team has decided to move the prediction related feature to 2.2.0, and we will GA 2.1.0 with the `History` source only. But `2.2.0-beta1` is not released yet. So maybe for now, just remove the dependency of PSReadLine.